vuepress/docs/next/tutorials/consume/multitenancy-tutorial.md

```diff
@@ -0,0 +1,248 @@
+
+Here is an example: **isn't part in yaml format and part in json format???**
+``` json
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: YOUR-TENANTS-CONFIGMAP
+data:
+  ENTANDO_TENANTS: >-
+        [
+            {
+                "tenantCode": "YOUR-TENANT1-CODE", # default subdomain name 
+                "fqdns": "YOUR-TENANT1-CODE.YOUR-DOMAIN" # value string and comma separated domains
+                "kcEnabled": true,
+                "kcAuthUrl": "http://YOUR-DOMAIN/auth",
+                "kcRealm": "YOUR-TENANT1-REALM",
+                "kcClientId": "quickstart",
+                "kcClientSecret": "YOUR-TENANT1-SECRET",
+                "kcPublicClientId": "entando-web",
+                "kcSecureUris": "",
+                "kcDefaultAuthorizations": "",
+                "dbDriverClassName": "org.postgresql.Driver",
+                "dbUrl": "YOUR-POSTGRESQL-TENANT1-URL",
+                "dbUsername": "postgres",
+                "dbPassword": "e7d60efa865c4510",
+                "cdsPublicUrl": "http://cds.YOUR-DOMAIN/YOUR-TENANT1-CODE/",
+                "cdsPrivateUrl": "http://YOUR-TENANT1-CODE-cds:8080/",
+                "cdsPath": "api/v1",
+                "solrAddress": "http://YOUR-SOLR-URL",
+                "solrCore": "TENANT1-CORE-NAME"
+            },
+            {
+                "tenantCode": "YOUR-TENANT2-CODE",
+                "fqdns": "www.YOUR-TENANT2-CODE.com,YOUR-TENANT2-CODE.YOUR-DOMAIN.com,news.YOUR-DOMAIN.com"
+                "kcEnabled": true,
+                "kcAuthUrl": "https://YOUR-DOMAIN/auth",
+                "kcRealm": "YOUR-TENANT2-REALM",
+                "kcClientId": "quickstart",
+                "kcClientSecret": "YOUR-TENANT2-SECRET",
+                "kcPublicClientId": "entando-web",
+                "kcSecureUris": "",
+                "kcDefaultAuthorizations": "",
+                "dbDriverClassName": "org.postgresql.Driver",
+                "dbUrl": "YOUR-POSTGRESQL-TENANT2-URL",
+                "dbUsername": "postgres",
+                "dbPassword": "xxx",
+                "cdsPublicUrl": "http://cds.YOUR-DOMAIN/YOUR-TENANT2-CODE/",,
+                "cdsPrivateUrl": "http://mt720-cds-YOUR-TENANT2-CODE-service.test-mt-720.svc.cluster.local:8080/",
+                "cdsPath": "api/v1",
+                "solrAddress": "http://solr-solrcloud-common.test-mt-720.svc.cluster.local/solr",
+                "solrCore": "TENANT2-CORE-NAME"
+  }
+        ]
+```
+
+2. Use kubectl to create a secret for the ConfigMap. Though this step is optional, it is recommended for best practices.
+```
+kubectl -n ent create secret generic tenant-config --from-file=ENTANDO_TENANTS=YOUR-TENANTS-CONFIGMAP
+```
+
+3. Scale down the App Engine deployment (de-app image) to 0
+4. Open the `entando-de-app` deployment and add the environment variable to point to the ConfigMap or to use the K8s Secret:  
```

Comment:
Are these options for adding the tenant ConfigMap correct? should I leave out any of them?